### PR TITLE
create first working version of the GPT2 embedding comparison

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary

Make an initial version to compare embeeding with GPT2

### CODE REVIEW

File: src/pds/llm/GPT2.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import pandas as pd
+
+
+tokenizer = GPT2Tokenizer.from_pretrained('gpt2')
+model = GPT2LMHeadModel.from_pretrained('gpt2')
+feature_extraction_pipeline = FeatureExtractionPipeline(model=model, tokenizer=tokenizer)
+
+
+def get_label_embeddings(url):
+    vectors = []
```
Comment (nutjob4life) [inline src/pds/llm/GPT2.py:16]: Docstring missing for this function; did `flake8` linting run for this commit?

**Review by nutjob4life — CHANGES_REQUESTED** [review]
Looks like some code linting issues … maybe the pre-commit hook didn't run? 🤔

File: src/pds/llm/GPT2.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
+from transformers import GPT2LMHeadModel, FeatureExtractionPipeline, GPT2Tokenizer
+
```
[the remark this answers is not in the document]
Comment (tloubrieu-jpl) [reply src/pds/llm/GPT2.py:2]: thanks @nutjob4life , I'm doing that

**Comment by nutjob4life** [comment]
> Next time, we would need to run `tox -e lint` to detect the style errors that we are committing to this repository, there are still plenty of them.

Also the pre-commit hooks should take care of that automatically. It's possible these commands didn't get run initially?
```
    pre-commit install
    pre-commit install -t pre-push
    pre-commit install -t prepare-commit-msg
    pre-commit install -t commit-msg
```

**Comment by tloubrieu-jpl** [comment]
Has been manually merged.

**Pull request closed without merging**